src/main/java/nextstep/ladder/domain/UserNames.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.ladder.domain;
+
+import java.util.List;
+
+public class UserNames {
+    private List<String> names;
+
+    public UserNames(List<String> names) {
+        checkNameLength(names);
+        this.names = names;
+    }
+
+    private void checkNameLength(List<String> stringNames) {
+        if(stringNames.stream().anyMatch(name -> name.length() > 5))
```

Comment:
어쩌면 구현하신 코드는 컴퓨터적인 언어아닐까요?
코드는 인간의 언어를 표현하는 것일까요 컴퓨터적인 표현을 하는것일까요?

```suggestion
        if(is_초과글자있어?(stringNames))
```

우리가 사용하는 말로 표현하면 어떠한 장점이 있을까요?

Reply:
음.. 우리가 사용하는 말로 표현하면, 코드가독성이 좋아질 것 같습니다!

또한, 메서드 구현내용을 볼 필요없이, 메소드명으로만 유추가능하겠네요